src/main/resources/assets/minecolonies/lang/en_us.lang

```diff
@@ -378,27 +383,22 @@ com.minecolonies.coremod.permission.shoot_arrow=Shoot arrows
 com.minecolonies.coremod.permission.attack_citizen=Attack citizens
 com.minecolonies.coremod.permission.attack_entity=Attack entities
 com.minecolonies.coremod.permission.access_free_blocks=Access free-blocks
-com.minecolonies.coremod.permission.teleport_to_colony=Teleport to Colony
 
 item.supplyCampDeployer.invalid=Requiring a flat ground of at least 16*17.
 item.minecolonies.supplyChestDeployer.name=SupplyShip Chest
 item.minecolonies.supplyCampDeployer.name=SupplyCamp Chest
 com.minecolonies.coremod.error.supplyChestAlreadyPlaced=You have already placed a supply ship or supply camp!
+com.minecolonies.coremod.gui.townHall.currentHappiness=Overall happiness:
 com.minecolonies.coremod.job.guard.toolClickGuardTooFar=Too far away from guard tower, please upgrade your guardtower.
+com.minecolonies.coremod.saturation.0=I'm starving, I can't work anymore
+com.minecolonies.coremod.saturation.3=I'm quite hungry, I feel very unproductive
+com.minecolonies.coremod.saturation.5=I'm hungry, I'm not that productive anymore
+com.minecolonies.coremod.saturation.7=I'm quite saturated, but I'll have to eat soon again
 
 com.minecolonies.coremod.item.permissionscepter.setmode=Set to mode '%s'
 com.minecolonies.coremod.item.permissionscepter.permission.deny=You do not have permission to edit blocks with this tool.
 com.minecolonies.coremod.item.permissionscepter.addblock.success=Added block type '%s'
 com.minecolonies.coremod.item.permissionscepter.addposition.success=Added location '%d %d %d'
 com.minecolonies.coremod.item.permissionscepter.removeblock.success=Removed block '%s'
 com.minecolonies.coremod.item.permissionscepter.removelocation.success=Removed location '%d %d %d'
-com.minecolonies.coremod.gui.townHall.currentHappiness=Overall happiness:
```

Comment:
Yes, and I think the baker things got lost here as well (you can add the baker to the statistic here as well)